web/src/cypress/support/schedule.ts

```diff
@@ -142,32 +145,48 @@ function setScheduleTarget(
 }
 
 function createSchedule(sched?: ScheduleOptions): Cypress.Chainable<Schedule> {
-  const query = `mutation createSchedule($input: CreateAllInput!){
-          createAll(input: $input) {
-            schedules {
-              id
-              name
-              description
-              timeZone: time_zone
-            }
-          }
-      }`
+  const query = `mutation createSchedule($input: CreateScheduleInput!){
+      createSchedule(input: $input) {
+        id
+        name
+        description
+        timeZone
+        isFavorite
+      }
+    }`
+
+  const favQuery = `mutation ($input: SetFavoriteInput!){
```

Comment:
Let's add this to the `createSchedule` mutation. We can make one call here, and then update the UI so new schedules are favorited by default like services and rotations.